### DISCUSSION TITLE
Simultaneous installation of Wireguard and OpenVPN

### DISCUSSION THREAD

**Discussion by aansari55** [angristan/wireguard-install#478] [Help]
Is it possible to install both Servieces WG and OV simultaneously from your scripts on one VPS? I have a cheap VPS with restricted resources (1Core CPU and 0.5 GB RAM). Could it work if I only want to have 5 or 6 active users?

**Comment by normalblaster** [comment]
I would say this is possible.  I have an RPi5 with the angristan scripted OpenVPN install and a long hand installed wireguard and they run side by side.
